Add guide on how to set trunk.io as default formatter in vscode (#379)
trunk-io/docs: code-quality/ide-integration/vscode.md

or click on the notification to "Share trunk config", which will commit `.trunk/trunk.yaml`, the Trunk configuration file.

### Trunk as default formatter

> Trunk does register as a formatter in vscode, remember that project should have Trunk enabled.

Reviewer: ```suggestion
You can use Trunk as your default formatter in VSCode if you have Trunk configured for the project.
```

You can set `trunk.io` as the default formatter for just one language as in the example, or as a default for all languages.

> In your settings json via:

Reviewer: ```suggestion

In your `settings.json` like this:
```
